### ISSUE TITLE
Copy feature and associated annotations from assembly to another

### ISSUE THREAD

**Issue by kyostiebi** [GMOD/Apollo3#118]
1. Create new change type _CopyFeaturesAndAnnotationsChange_.
2. Create new interface containing the following properties: _assemblyId, targetAssemblyId, featureId_
3. The new change  _CopyFeaturesAndAnnotationsChange_ copies given feature (and all children) + associated annotations into target assembly **with new generated featureIds**. _refSeqIds_ will be refSeqIds of targetAssembly.

_"Copy genes or transcripts from one gene set to another. Also copies all associated functional annotations."
Basically this would involve a new change type that adds a single top-level feature from the data in a POST request. Then the UI could call this endpoint for e.g Assembly2 with a copy of the data from a feature on Assembly1. Then once that endpoint works, we can create a UI action for it, although that step might be better as a separate issue."_
